--- origsrc/libxslt-1.1.28/tests/plugins/Makefile.am	2012-11-20 21:08:02.000000000 -0600
+++ src/libxslt-1.1.28/tests/plugins/Makefile.am	2015-03-18 18:41:14.493458100 -0500
@@ -21,7 +21,7 @@ plugindir = $(abs_builddir)/.libs
 xmlsoft_org_xslt_testplugin_la_CFLAGS = -DMODULE_COMPILE $(LIBXML_CFLAGS) $(LIBXSLT_CFLAGS)
 xmlsoft_org_xslt_testplugin_la_SOURCES = testplugin.c
 xmlsoft_org_xslt_testplugin_la_LIBADD = $(top_builddir)/libxslt/libxslt.la $(EXTRA_LIBS)
-xmlsoft_org_xslt_testplugin_la_LDFLAGS = -module -avoid-version -rpath $(plugindir)
+xmlsoft_org_xslt_testplugin_la_LDFLAGS = -module -avoid-version -no-undefined -rpath $(plugindir)
 
 test-logall:
 	@echo '## Running plugin tests'
